src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -316,6 +316,43 @@ internal static SemanticVersion PSV6Version
     }
 
     /// <summary>
+    /// Represents an implementation of '$PSVersionTable' variable.
+    /// The implementation contains ordered 'Keys' and 'GetEnumerator' to get user-friendly output.
+    /// </summary>
+    public sealed class PSVersionHashTable : Hashtable, IEnumerable
+    {
+        internal PSVersionHashTable(IEqualityComparer equalityComparer) : base(equalityComparer)
+        {
+        }
+
+        /// <summary>
+        /// Returns ordered collection with Keys of 'PSVersionHashTable'
+        /// </summary>
+        public override ICollection Keys
+        {
+            get
+            {
+                Array arr = new string[base.Keys.Count];
+                base.Keys.CopyTo(arr, 0);
+                Array.Sort(arr, StringComparer.OrdinalIgnoreCase);
```

Comment:
We have several open discussions on adding additional rows. Alphabetical order will be even more useful after that.

If you want only PSVersion why don't use `$PSVersionTable.PSVersion`?

I wonder how it makes Windows PowerShell for `PSVersionHashTable` _hashtable_ ?

I could slightly change this code to move PSVersion in the first place without moving to `OrderedDictionary`